### DISCUSSION TITLE
Can't use canvas

### DISCUSSION THREAD

**Discussion by dinkel1** [thatskymod/Sky-CotL-Scripts#60] [General]
Hello guys, im new to be dukun, i ask my friend how to be dukun, she give me canvas and this link, but my canvas didn't work, the mod manager didn't show, how to fix this guys🙏

**Comment by thatskymod** [comment]
It seems that there are several issues at play here. You mentioned being new to modding, and you've also stated that your Canvas app is not behaving as expected.

To help us better understand the problem, could you specify what exactly you expected to happen that isn't occurring? Is the issue with Canvas itself, or is it possible that you overlooked the 'Mod Manager' icon in your app drawer?

For a good starting point, have you followed the steps outlined at https://github.com/thatskymod/Sky-CotL-Scripts/tree/main/Canvas#getting-started?

**Comment by thatskymod** [comment]
Closing this due to the lack of activity and insufficient information from the author.
